cmd/uplink/cmd/ls.go

```diff
@@ -26,7 +27,8 @@ func init() {
 		Short: "List objects and prefixes or all buckets",
 		RunE:  list,
 	}, RootCmd)
-	recursiveFlag = lsCmd.Flags().Bool("recursive", false, "if true, list recursively")
+	lsRecursiveFlag = lsCmd.Flags().Bool("recursive", false, "if true, list recursively")
+	lsEncryptedFlag = lsCmd.Flags().Bool("encrypted", false, "if true, show paths as base64-encoded encrypted paths")
```

Comment:
Why do we need an extra flag when we already have `enc.path-type`?

Reply:
Using `enc.path-type` won't have any effect on a list or delete operation.

Reply:
Isn't `enc.path-type` used only when creating a new bucket?

Reply:
Perhaps, we can rename `--encrypted` to `--encrypted-paths` to be clearer that is related to paths only.

Reply:
One option is to have a separate command entirely as well `ls-encrypted` or something.

Reply:
I believe that's correct. The only place I see it used in uplink is in `GetPathCipherSuite` (uplink/config.go:149) and that's used in `makeBucket` (uplink/cmd/mb.go:58).